docs/source/tutorial/short.rst

```diff
@@ -0,0 +1,9 @@
+.. _tutorial:short:
```

Comment:
I'm wondering a little bit about the name `short` here, since it will appear in the url. Maybe `index.rst` is better?